infrastructure/generators/native.py

```diff
@@ -18,7 +18,7 @@
 ]
 
 
-def native_data_generator(**event_kwargs):
+def native_data_generator(num_frames=500):
```

Comment:
That's pretty high for a default, isn't it?

Reply:
fixed